MotorPair.cpp

```diff
@@ -0,0 +1,22 @@
+#include "MotorPair.h"
+
+
+
+MotorPair::MotorPair()
+{
+	left = new motor();
+	right = new motor();
+}
+
+
+MotorPair::~MotorPair()
+{
+	delete left, right;
```

Comment:
this does not do what you think it does;
it is the equivalent of
``` C++
delete left;
right; // this has no effect
```
right is not deleted